runelite-client/src/main/java/net/runelite/client/config/ConfigManager.java

```diff
@@ -468,8 +468,31 @@ public ConfigDescriptor getConfigDescriptor(Config configurationProxy)
 			throw new IllegalArgumentException("Not a config group");
 		}
 
+		final List<ConfigSectionDescriptor> sections = Arrays.stream(inter.getDeclaredFields())
+			.filter(m -> m.isAnnotationPresent(ConfigSection.class) && m.getType() == String.class)
+			.map(m ->
+			{
+				try
+				{
+					return new ConfigSectionDescriptor(
+						String.valueOf(m.get(inter)),
+						m.getDeclaredAnnotation(ConfigSection.class)
+					);
+				}
+				catch (IllegalAccessException e)
+				{
+					return null;
+				}
```

Comment:
Due to having the keyName be the contents of the field that the annotation is attached to, I have to reflect to get it. If I don't have this try/catch it won't compile since it can throw, and if it does throw then it's not something we'd need. If you have a cleaner way to do this I would definitely love to know.

Reply:
I see, that's unfortunate. I maintain we should log for failures here, even if we don't expect them. I'd rather log some warnings than have silent failures in the event something went wrong.